features/admin.administrators.v1/wizard/add-administrator-wizard.tsx

```diff
@@ -242,10 +248,28 @@ export const AddAdministratorWizard: FunctionComponent<AddUserWizardPropsInterfa
             setIsSubmitting(false);
             closeWizard();
         } else {
-            // If not, prompt to assign the admin role to the user.
-            setIsSubmitting(false);
-            setShowAdminRoleAddConfirmationModal(true);
-            setSelectedUser(originalAdminUserList.Resources[ 0 ]);
+            const assignAdminRoleToExistingPrimaryUser: boolean =
```

Comment:
Can we remove this constant since this has one usage?

Reply:
Fixed with 8ea0404